gef-ui/templates/includes/header.njk

```diff
@@ -1,42 +1,12 @@
-{% from "govuk/components/header/macro.njk" import govukHeader %}
+{% import '../partials/header-component.njk' as headerComponent %}
+{% import '../partials/header-component-no-user.njk' as headerComponentNoUser %}
 
 {% if user %}
 
-{# set profileLink = urlencode(user._id) #}
-
-{{ govukHeader({
-  homepageUrl: "https://www.gov.uk",
-  serviceName: "Get a guarantee for export finance",
-  serviceUrl: "/dashboard",
-  navigation: [
-    {
-      href: "/user/" + user._id,
-      text: user.username,
-      active: true
-    },
-    {
-      href: "/user/" + user._id,
-      text: "Profile"
-    },
-    {
-      href: "/logout",
-      text: "Sign out"
-    }
-  ],
-  attributes: {
-    "data-cy": "header"
-  }
-}) }}
+  {{headerComponent.render(user)}}
```

Comment:
the header component should be a `partial`. we don't need to pass variables to the partial because it picks them up automatically from the parent - meaning from here

Reply:
removed